### PR TITLE
Settle buffer for receiving ball

### PR DESCRIPTION
**Pull request by HussainGynai**
The receive action should stop settling after the robot has the ball for at least 0.25 seconds (based this off of 60 Hz) or the ball slows down enough.

### CODE REVIEW

File: rj_gameplay/rj_gameplay/action/receive.py [hunk 1]
```diff
@@ -28,9 +29,12 @@ def tick(self, intent) -> None:
         return intent
 
     def is_done(self, world_state) -> bool:
+        if world_state.our_robots[self.robot_id].has_ball_sense:
+            self.has_ball_ticks += 1
+        print(self.has_ball_ticks)
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/action/receive.py]:
```suggestion
```

File: rj_gameplay/rj_gameplay/action/receive.py [hunk 2]
```diff
@@ -28,9 +29,12 @@ def tick(self, intent) -> None:
...
         if self.robot_id is None:
             return False
         #TODO: Use local params for this threshold
-        if world_state.our_robots[self.robot_id].has_ball_sense or np.linalg.norm(world_state.ball.vel) < SETTLE_BALL_SPEED_THRESHOLD:
+        if self.has_ball_ticks >= 15 or np.linalg.norm(world_state.ball.vel) < SETTLE_BALL_SPEED_THRESHOLD:
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/action/receive.py]: I think you should reset has_ball_ticks here

**Review by kfu02 — CHANGES_REQUESTED** [review]
two comments, good to merge after

File: rj_gameplay/rj_gameplay/action/receive.py (hunk 2 above)
Comment (HussainGynai) [reply rj_gameplay/rj_gameplay/action/receive.py]: Ah yeah forgot about that lol

**Pull request merged**